src/local/README.md

```diff
@@ -20,3 +20,23 @@ This script requires [jib-cli](https://github.com/GoogleContainerTools/jib/tree/
 Usage: `create-containers.sh [version] [jre-version]`
 * `version` is the dataflow-server version like `2.9.6`. Default is `2.11.3-SNAPSHOT`
 * `jre-version` should be one of 11, 17. Default is 11
+
+
+## Testing with Keycloak
+### Build with Java 17
+```shell
+./mvnw clean install -am -pl :spring-cloud-dataflow-server -DskipTests -Dmaven.javadoc.skip=true -T 1C
+```
+### Execute:
+```shell
+./src/local/launch-keycloak.sh
+```
+
+### Debug
+Module: `spring-cloud-dataflow-server`
+Class: `org.springframework.cloud.dataflow.server.single.DataFlowServerApplication`
+Arguments: `--spring.cloud.dataflow.features.streams-enabled=false --spring.cloud.dataflow.features.tasks-enabled=true --spring.cloud.dataflow.features.schedules-enabled=false --spring.config.additional-location="$ProjectFileDir$/src/local/application-dataflow-keycloak.yaml"`
```

Comment:
Just curious as to why we are disabling streams?

Reply:
This was for testing login when only running Keycloak and Data Flow Server.